src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,9 +19,13 @@ public User login(String login, String password) throws AuthenticationException
     }
 
     private User findByLogin(String login) {
+        log.debug("Service level. Find by login was called with params: {}", login);
         User user = new User(login, "1234");
         // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        log.debug("Service level. Found user by id: {}, user's data {}",
```

Comment:
This log message is informative, but ensure that logging user's data does not expose sensitive information. If 'user' contains sensitive data, consider logging only non-sensitive information.